docs/earth-relief.rst

```diff
@@ -60,7 +60,7 @@ do not specify a CPT then this dataset default to the GMT master *geo*.
 Technical Information
 ~~~~~~~~~~~~~~~~~~~~~
 
-As you see, the 30s and lower resolutions are all derivatives of Scripps' SRTM15+V2.3 grid
+As you see, the 30s and lower resolutions are all derivatives of Scripps' SRTM15+V2.4 grid
 (Sandwell et al., 2022).  We have downsampled it via Cartesian Gaussian filtering to prevent
```

Comment:
Should this cite change to Tozer et al. 2019?

Reply:
Yes, Paul checked and this should be Tozer et al., 2019. I will push an update.